### PR TITLE
Loading screen Update and CutScene Fixes

### PR DESCRIPTION
**Pull request by accousins**
Loading Screen now has the logo, and the … scrolls.  It is a also a
prefab, so updating it once changes them all.  CutScenes should now
scale across devices.

### CODE REVIEW

File: TuckersTemple/Assets/Scripts/CutSceneManager.cs [hunk 1]
```diff
@@ -104,7 +104,7 @@ public class CutSceneManager : MonoBehaviour {
 		}
 
 		//set background to first image, and make the text box invisible
-		background.GetComponent<SpriteRenderer> ().sprite = cutScenes[0].BackgroundImage;
+		background.GetComponent<Image> ().sprite = cutScenes[0].BackgroundImage;
```
Comment (accousins) [inline TuckersTemple/Assets/Scripts/CutSceneManager.cs:107]: This is right

**Review by accousins — COMMENTED** [review]
Elliot made me do it

File: TuckersTemple/Assets/Scripts/CutSceneManager.cs [hunk 2]
```diff
@@ -159,7 +159,7 @@ public class CutSceneManager : MonoBehaviour {
 		sceneIndex++;
 		lineIndex = -1; //reset line index
 		//load in the new image
-		background.GetComponent<SpriteRenderer> ().sprite = cutScenes [sceneIndex].BackgroundImage;
+		background.GetComponent<Image> ().sprite = cutScenes [sceneIndex].BackgroundImage;
```
Comment (accousins) [inline TuckersTemple/Assets/Scripts/CutSceneManager.cs:162]: do it

File: TuckersTemple/Library/LastSceneManagerSetup.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 sceneSetups:
-- path: Assets/Scenes/main.unity
+- path: Assets/Scenes/mainMenu.unity
```
Comment (accousins) [inline TuckersTemple/Library/LastSceneManagerSetup.txt:2]: Start this here

**Review by accousins — COMMENTED** [review]
I don't know whats happening

**Review by mcwillikers — APPROVED** [review]

**Pull request merged**